quarkus/config-api/src/main/java/org/keycloak/config/SecurityOptions.java

```diff
@@ -18,6 +18,17 @@ public class SecurityOptions {
             .defaultValue(FipsMode.DISABLED)
             .build();
 
+    public static final Option<String> TRUSTSTORE = new OptionBuilder<>("truststore", String.class)
+            .category(OptionCategory.SECURITY)
+            .description("List of pkcs12 (.p12 or .pfx) or PEM files that will be used to construct the javax.net.ssl.trustStore used at runtime.")
```

Comment:
I probably would not emphasise the `javax.net.ssl.trustStore` property here. It's an implementation detail and might be confusing to non-Java savvy users. Maybe something like this instead?
```suggestion
            .description("List of pkcs12 (.p12 or .pfx) or PEM files that will be used as a global truststore.")
```
We could then go into more details in docs, incl. mention that it uses `javax.net.ssl.trustStore`.
WDYT?

Reply:
Sounds good.